queries/prql/highlights.scm

```diff
@@ -0,0 +1,140 @@
+[
+  (keyword_from)
+  (keyword_filter)
+  (keyword_derive)
+  (keyword_group)
+  (keyword_aggregate)
+  (keyword_sort)
+  (keyword_take)
+  (keyword_window)
+  (keyword_join)
+  (keyword_select)
+  (keyword_switch)
+  (keyword_inner)
+  (keyword_right)
+  (keyword_full)
+  (keyword_in)
+  (keyword_rolling)
+  (keyword_rows)
+  (keyword_expanding)
+  (keyword_let)
+  (keyword_prql)
+  (keyword_switch)
+  (keyword_append)
+  (keyword_remove)
+  (keyword_intersect)
+] @keyword
+
+(literal) @string
+
+(assignment
+  alias: (field) @field)
+
+alias: (identifier) @field
+
+(f_string) @string.special
+(s_string) @string.special
+
+(comment) @comment
+
+(keyword_func) @keyword.function
+
+(function_call
+  (identifier) @function.call)
+
+[
+  "+"
+  "-"
+  "*"
+  "/"
+  "="
+  "=="
+  "<"
+  "<="
+  "!="
+  ">="
+  ">"
+  "->"
+  (bang)
+] @operator
+
+[
+  "("
+  ")"
+  "["
+  "]"
+] @punctuation.bracket
+
+[
+  ","
+  "."
+  (pipe)
+] @punctuation.delimiter
+
+(literal
+  (integer) @number)
+
+(integer) @number
+
+(literal
+  (decimal_number) @float)
+
+(decimal_number) @float
+
+[
+  (keyword_min)
+  (keyword_max)
+  (keyword_count)
+  (keyword_count_distinct)
+  (keyword_average)
+  (keyword_avg)
+  (keyword_sum)
+  (keyword_stddev)
+  (keyword_count)
+] @function.call
+
+[
+ (keyword_side)
+ (keyword_version)
+ (keyword_target)
+ (keyword_null)
+ (keyword_format)
+] @attribute
+
+(target) @function.builtin
+
+ [
+  (date)
+  (time)
+  (timestamp)
+] @string.special
+
+[
+  (keyword_left)
+  (keyword_inner)
+  (keyword_right)
+  (keyword_full)
+  (keyword_csv)
+  (keyword_json)
+] @method
```

Comment:
I don't think `@method` applies here.

Reply:
Yeah, I know that `@method` is meant for things like class methods, but this language is fairly simplistic. In this case it refers to the join method or the "parse that file" method.

Reply:
I'm still not convinced we should use `@method` here. What does SQL use?

Reply:
Consider this example:
```
from_text format:csv """
a,b,c
1,2,3
"""

from invoices
join side:left employees [==id]
join side:inner employees [==id]
join side:right employees [==id]
```
Keywords are `from_text`, `from` and `join` while `csv` or the join type are always instruction on how to perform the previous action (e.g. from_text or join)
Therefore I find the label `@method` more fitting than `keyword`

SQL is a bad example. I did a cleanup of these queries last week because almost everything was a keyword, hence some statements consists of uniformly colored words.

Reply:
Maybe `@method.call` then? Aren't you `calling` json on the table/data?

Reply:
Yes, that could be an alternative. I will change it later this day.

Reply:
@ObserverOfTime